src/lib/annotations/image/ImageAnnotator.js

```diff
@@ -105,7 +105,7 @@ class ImageAnnotator extends Annotator {
             type
         };
 
-        if (!annotatorUtil.validateThreadParams(threadParams)) {
+        if (annotatorUtil.validateThreadParams(threadParams)) {
```

Comment:
Does this mean that if `validateThreadParams()` returns `true`, that it isn't valid?